build.gradle

```diff
@@ -26,18 +27,30 @@ allprojects {
     repositories {
         mavenLocal()
     }
-}
 
-subprojects {
     apply plugin: 'java'
 
     java.toolchain.languageVersion.set(JavaLanguageVersion.of(project.java_version))
+
+    apply plugin: 'net.neoforged.neodev'
 }
 
 repositories {
     mavenCentral()
 }
 
+// Remove src/ sources from the root project. They are used in the neoforge subproject.
+sourceSets {
+    main {
+        java {
+            srcDirs = []
+        }
+        resources {
+            srcDirs = []
+        }
+    }
+}
+
```

Comment:
Looks like it wasn't necessary because the java plugin wasn't getting applied to the root project? I'm not sure why I had to apply java, I think it's for the neodev plugin. We shouldn't need the neodev plugin for the root project though.